docs/help/_posts/2015-01-01-how-nsub-works.markdown

```diff
@@ -66,6 +66,6 @@ The final thing to notice here is that there is the potential for real logic fro
 * Be careful substituting for classes!
 * Where possible use interfaces instead.
 * Remember NSubstitute works by inheriting from (or implementing) your original type. If you can't override a member by manually writing a sub-class, then NSubstitute won't be able to either!
-* Install [NSubstitute.Analyzers](/help/nsubstitute-analyzers) where ever you install NSubstitute. This will help you avoid these (and other) pitfalls.
```

Comment:
maybe it's better to rename the file instead? to be consistent with the name

Reply:
It would be better, but I already have links to the uri in SO answers. 😓